apps/web/src/App.css

```
@@ -3,7 +3,7 @@
   @apply bg-muted;
   transition: background-color 300ms ease;
 }
-@import "@repo/ui/globals.css";
+@import '@repo/ui/globals.css';
```

Stylelint (17.14.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)